### PR TITLE
Fix #741 - update README

### LINKED ISSUE #741: Make sure README is up-to-date
Please close if it is current; I'm not sure

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

**Comment by humphd** [comment]
see https://github.com/humphd/thimble.webmaker.org/blob/362f2aa68c14e2d72372bcf30454112c88b7fe4f/README.md for rendered version

File: README.md [hunk 1]
```diff
@@ -34,6 +39,8 @@ You can now run Thimble from the thimble.webmaker.org directory using
 
 ```node app```
```
Comment (gideonthomas) [inline README.md]: Let's use ```npm start``` instead

File: README.md [hunk 2]
```diff
@@ -34,6 +39,8 @@ You can now run Thimble from the thimble.webmaker.org directory using
```
Comment (sedge) [inline README.md]: I can't comment on it, but we skip 5 above

File: README.md [hunk 3]
```diff
@@ -63,9 +70,7 @@ be good to go.
 Building the front-end
 ----------------------
 
-Run `grunt build` to regenerate the front-end logic and css files. For production,
-run with `grunt build:prod` to get minified versions, and run `grunt build:prod:true` to enable
-auto building whenever a front-end file changes. See `Gruntfile.js` for details.
+Run `npm postinstall` to download and regenerate the front-end. See `Gruntfile.js` for details.
```
Comment (sedge) [inline README.md]: This is wrong. The `postinstall` hook means running `npm install` will kick off that script. They could also use `grunt requirejs`

File: README.md [hunk 4]
```diff
@@ -74,13 +79,15 @@ the latest state of development.
...
+In order to load Thimble, the Bramble code editor must be statically served on a separate domain.
 The simplest way to do this is to:
 
-1) Clone the editor (outside your cloned Thimble source)
+1) Clone the editor (outside your cloned Thimble source) and use the `bramble` branch
 
 ```
 git clone https://github.com/humphd/brackets.git --recursive
+cd brackets
+git checkout bramble
```
Comment (gideonthomas) [inline README.md]: We don't need this because it is the default branch

File: README.md (hunk 3 above)
Comment (sedge) [inline README.md]: **kick off that script if installation succeeds

**Comment by sedge** [comment]
This isn't bad. Should we mention the other external deps? publish & id?

**Comment by sedge** [comment]
Ugh, and s3 cuz local s3 faking is broken atm.

File: README.md (hunk 3 above)
Comment (sedge) [inline README.md]: *coughs politely*

**Comment by sedge** [comment]
one nit then r+

File: README.md [hunk 5]
```diff
@@ -1,53 +1,121 @@
...
+* Run ``npm start`` to start the server
+
+**login.webmaker.org**
+* Clone https://github.com/mozilla/login.webmaker.org
+* Run ``npm install`` to install dependencies
+* Run ``cp env.sample .env`` to create an environment file
+* Run ``npm start`` the server
+
+**PostgreSQL**
+* Install Postgres with via Homebrew
```
Comment (gideonthomas) [inline README.md]: Install Postgres via Homebrew

File: README.md [hunk 6]
```diff
@@ -1,53 +1,121 @@
...
+* Go to [http://localhost:3000/account](http://localhost:3000/account)
+* Click ``Join Webmaker`` and complete the process, you can use a fake email
+* When you've created your account, click ``Set permanent password instead``
+  * This lets you authenticate your account without needing email
+* Go back to Thimble and Log In with your new account
+
+## Running the parts
+This is the list of commands to get each part up and running.
+
+* Thimble ``node app``
```
Comment (gideonthomas) [inline README.md]: ``npm start``

File: README.md [hunk 7]
```diff
@@ -1,53 +1,121 @@
...
+* Brackets ``npm start``
+* PostgreSQL Database ``postgres -D /usr/local/var/postgres``
+* Webmaker ID server ``npm start``
+* Webmaker Login Server ``npm start``
+* Webmaker Publishing Server ``npm start``
+* Local publish folder ``cd /tmp/mox && http-server -p 8001``
```
Comment (gideonthomas) [inline README.md]: ``cd /tmp/mox/test && http-server -p 8001``

**Comment by gideonthomas** [comment]
few comments, r=me with that

**Pull request merged**